test/verify/check-storage-luks

```diff
@@ -153,6 +154,8 @@ class TestStorage(StorageCase):
         #
         #        https://github.com/latchset/tang/issues/23
 
+        m.execute("systemctl restart tangd.socket")
```

Comment:
Hmm, this looks odd here.

Reply:
Yes, I think there was some issue on my side and I added this line there.

Reply:
I have removed it now, let's see if the tests tell us anything concrete.